cmd/openshift-tests/disaster-recovery.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	"k8s.io/kubernetes/pkg/kubectl/util/templates"
+
+	"github.com/openshift/origin/pkg/test/ginkgo"
+	_ "github.com/openshift/origin/test/e2e/dr"
+)
+
+// disasterRecoverySuites are all known disaster recovery test suites this binary should run
+var disasterRecoverySuites = []*ginkgo.TestSuite{
```

Comment:
I would expect these tests to be in the disruptive suite and run by it.

Reply:
A new suite was created for a simple cmdline to run each test as these take ~30 min to complete each.

Disruptive suite would run these tests too.

Reply:
> I would expect these tests to be in the disruptive suite and run by it.

I'm convinced that it's worth a separate suite for disaster recovery.  It's a long suite, there are special skip rules, and the goal is to have a disaster, recover it, and prove we can run normal tests after.  That last bit is important for proof and I don't think disruptive does it.

Reply:
There are zero things in disruptive today.  You would be the first disruptive.  If we need to reduce time to add more critical disruptive we would.

1h for a suite is fine.

Reply:
To run an individual test is easy, so having a suite for them is not required.

Reply:
These suites would run a single test, then e2e test with several tests skipped. It could be run separately, however that would only complicate a command for CI test

Reply:
That's not how the structure of these commands works.  "Running all e2e after disruption" is a different kind of test that you can't do within this command today.

Reply:
There is no value in a dedicated command for that though, especially since we have to run `/parallel/dr` with a few skipped tests. Isn't it the way `openshift-tests run-upgrade` works?

Although if @deads2k agrees I can rework it and make it run